### ISSUE TITLE
Roadmap?

### ISSUE THREAD

**Issue by dubcanada** [Draggable/formeo#46]
Hello,

I want to use this for a project and I was wondering if you had a roadmap of ideas until 1.0? I'd like to help where I can, but I don't see any sort of list of things to be done/are done.

Thanks

**Comment by derrandz** [comment]
:+1: 

We should probably think about that @kevinchappell

**Comment by kevinchappell** [comment]
Agreed. A proper roadmap is needed. Will reference this issue when pushed.

**Comment by onyekaa** [comment]
I'd like to help as well. I recently had to use this plugin because it's the only form-builder I could find that one could arrange in columns, and made a few modifications to fill functionality gaps so it would be nice to do that properly.
